frontend/packages/console-shared/src/components/form-utils/FormFooter.tsx

```diff
@@ -22,65 +23,74 @@ const FormFooter: React.FC<FormFooterProps> = ({
   hideSubmit = false,
   showAlert,
   sticky,
+  shadow = true,
```

Comment:
Shouldn't all sticky footers have a shadow?

Best practice is to have boolean properties default to false because it allows for the shorthand jsx notation to set to true.

Reply:
Perhaps this is present because of the issue you have with the pipeline builder form?

Reply:
correct. currently pipeline and helm page layout is not fixed.. that's why.
I was going to remove the shadow property altogether from the component.
Do we need a shadow prop?